app/controllers/plays_controller.rb

```diff
@@ -6,10 +6,12 @@ class PlaysController < ApplicationController
 
   def index
     authorize Play
-    @plays = apply_scopes(policy_scope(Play))
-             .paginate(page: params[:page], per_page: params[:per_page])
+    scoped_plays = apply_scopes(policy_scope(Play))
+    @plays = scoped_plays.paginate(page: params[:page], per_page: params[:per_page])
     add_pagination_headers(@plays)
 
+    return unless stale?(etag: [scoped_plays.cache_key, scoped_plays.cache_version(:played_at)])
```

Comment:
Why not add the timestamps for plays as well so that we can keep the logic the same everywhere? I honestly don't know anymore what the motivation was for leaving off the timestamps on some models; probably some misguided idea about the efficiency of not storing data that's not needed.